src/main/webapp/project/NewProjectDocuments.js

```diff
@@ -75,6 +82,16 @@ export const NewProjectDocuments = hh(class NewProjectDocuments extends Componen
     });
   };
 
+  handleAttestationCheck = (e) => {
+    let value = e.target.checked;
```

Comment:
this could be a const ... it's not getting modified anywhere.

Reply:
You are totally right. I'll change this.